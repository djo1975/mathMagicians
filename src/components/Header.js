import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
  <header>
    <h1>Math magicians</h1>
    <nav>
      <ul style={{ listStyle: 'none', display: 'flex' }}>
        <li style={{ marginRight: '10px', paddingRight: '10px', borderRight: '2px solid rgb(0, 0, 255)' }}>
          <Link to="/">Home</Link>
        </li>
        <li style={{ marginRight: '10px', paddingRight: '10px', borderRight: '2px solid rgb(0, 0, 255)' }}>
          <Link to="/calculator">Calculator</Link>
        </li>
        <li>
          <Link to="/quotes">Quotes</Link>
        </li>
      </ul>
    </nav>
  </header>

);

export default Header;
