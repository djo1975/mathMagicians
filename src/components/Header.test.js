import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';
import Header from './Header';

Enzyme.configure({ adapter: new Adapter() });

describe('Header', () => {
  it('should renders correctly', () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
