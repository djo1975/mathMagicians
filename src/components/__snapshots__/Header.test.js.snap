// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should renders correctly 1`] = `ShallowWrapper {}`;
